contrib/ledger-tests/solcap.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Can we add a cli flag for verbosity levels.  A lot of the transaction information gets printed with -v 4